### PR TITLE
libuv: fix shared lib name on windows

### PR DESCRIPTION
**Pull request by paul-reilly**
`uv` is the .lib that is associated with the DLL for libuv.

### CODE REVIEW

**Comment by waruqi** [comment]
>  /unix/thread.c:399: undefined reference to `pthread_once

maybe we need add syslinks(pyhread) for linux.

**Comment by paul-reilly** [comment]
Yes, that was the one!

File: packages/l/libuv/xmake.lua [hunk 1]
```diff
@@ -28,6 +28,10 @@ package("libuv")
         package:add("syslinks", "advapi32", "iphlpapi", "psapi", "user32", "userenv", "ws2_32", "kernel32", "gdi32", "winspool", "shell32", "ole32", "oleaut32", "uuid", "comdlg32")
     end)
 
+    on_load("linux", function (package)
+        package:add("syslinks", "pthread")
+    end)
```
Comment (waruqi) [inline packages/l/libuv/xmake.lua:33]: we need only use add_syslinks()

```lua
if is_plat("linux") then
    add_syslinks("pthread")
end
```

**Comment by waruqi** [comment]
In addition, in order to reduce the commit history, please merge into a single commit. Thanks.

**Pull request closed without merging**